Add sleepy_craft_studios sleepy_keeb (#23659)
qmk/qmk_firmware: keyboards/sleepy_craft_studios/sleepy_planck/readme.md

> # sleepy_planck

Reviewer: ```suggestion
# Sleepy Planck
```

> ![sleepy_planck](https://bucket.sleepycraftstudios.com/images/sleepy_planck_complete_blue_silver_silk-rotated-270.webp)

Reviewer: Please can this image be (re-)hosted on Imgur?

Author: Sure thing

The Sleepy Planck is a hand-wired keyboard based on the 4x12 Planck layout. The Sleepy Planck integrates hot-swap socket and diode holders into the plate for easier building.

- Keyboard Maintainer: [Sleepy Craft Studios](https://github.com/sleepy-craft-studios)

> - Hardware Supported: Pro-Micro and clones, Elite-C

Reviewer: ```suggestion
- Hardware Supported: Pro-Micro (and clones/adapations of)
```

> - Pro Micro development board (and clones/adapations of)

Reviewer: ```suggestion
- Hardware Supported: Pro Micro development board (and clones/adapations of)
```

Author: oops, fixed.
